nemo/collections/nlp/models/lm_model.py

```diff
@@ -154,7 +158,8 @@ def validation_step(self, batch, batch_idx):
 
         loss = self.agg_loss(loss_1=mlm_loss, loss_2=nsp_loss)
 
-        tensorboard_logs = {'val_loss': loss}
+        perplexity = self.perplexity(mlm_loss)
```

Comment:
how about changing the name of var perplexity to perp_val or the name of perplexity to perplexity_metric?

Reply:
i like perplexity_metric